tests/terratest/functions/test/forceCleanup.go

```diff
@@ -0,0 +1,18 @@
+package functions
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	cleanup "github.com/rancher/rancher/tests/terratest/functions/cleanup"
+)
+
+func ForceCleanup(t *testing.T) {
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../../../../terratest/modules/cluster",
```

Comment:
Earlier suggestion on the directory path can potentially be applied here as well.

Reply:
Please help me understand the concern.  I will implement the suggestion, but i'd like to understand how its superior or an improvement to what has already been implemented.  What is concerning??

Reply:
Copying/pasting from prior comment 😄 

More of an efficient method as opposed to a concern. I don't think Terratest will have this issue, but in the past, Jenkins has not played nicely with absolute paths. This is a way that has a guaranteed method to work across environments when integrating with Jenkins.